docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+    cinema:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            -   ./:/app
+            - my_media:/files/media
+        command: >
+            sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+        env_file:
+            - .env
+        depends_on:
+            - db
+
+    db:
+        image: postgres:14-alpine
+        ports:
+            - "5433:5432"
```

Comment:
The database port mapping uses `5433:5432`. Ensure that port `5433` is intended for use on the host machine and does not conflict with other services. Typically, `5432:5432` is used unless there is a specific reason to change it.